packages/mds-types/index.ts

```diff
@@ -25,8 +25,14 @@ export const Enum = <T extends string>(...keys: T[]) =>
 export const isEnum = (enums: { [key: string]: string }, value: unknown) =>
   typeof value === 'string' && typeof enums === 'object' && enums[value] === value
 
-export const VEHICLE_TYPES = Enum('car', 'bicycle', 'scooter', 'moped', 'recumbent')
-export type VEHICLE_TYPE = keyof typeof VEHICLE_TYPES
+export const MICRO_VEHICLE_TYPES = ['car', 'bicycle', 'scooter', 'moped', 'recumbent'] as const
```

Comment:
I know it is preexisting, but is `car` really considered a micromobility type? If so, is it used? f not, it would obviate the need for a new `taxi` type.

Reply:
I believe it is, some providers are using the micromobility state machine for 'carshare' today (e.g. BlueLA), even though I'd argue that the operational behavior of carshare isn't _entirely_ the same as MM. 

The actual solution to this is that we need to add a notion of `mode` for vehicles. Will be adding that in the next few weeks as there are more tweaks made to support the Taxi model.